### PR TITLE
Update the docker image links.

### PR DESCRIPTION
**Pull request by Thelta**
When I try the old links (`proget.makedeb.org/docker/` and `proget.hunterwittenborn.com/docker`) in a Dockerfile, docker couldn't pull the image (gives 403 forbidden). After noticing the images has not been updated since 2023/06, I noticed docker images moved to github packages.

### CODE REVIEW

**Comment by hwittenborn** [comment]
How are you pulling the images @Thelta? I'm able to pull everything just fine on my end.

The Docker images are still on the ProGet instance, but those shouldn't be the ones that end up getting used (I've been meaning to delete them). I've got a proxy set up that automatically redirects Docker traffic from `proget.makedeb.org/docker` and `proget.hunterwittenborn.com/docker` to `ghcr.io`.

Regardless of that, I've been wanting this to get done for a while. I have the proxy set up for people who need the old URLs, but GitHub Packages is where I'm wanting new users to start getting makedeb's images from.

Thanks for getting this done, but would you be available to discuss the issue with no being able to pull from the ProGet URLs? We could discuss it here or in makedeb's support rooms if you'd be open to that.

**Review by hwittenborn — APPROVED** [review]

**Comment by Thelta** [comment]
> How are you pulling the images @Thelta? I'm able to pull everything just fine on my end.
> 
> The Docker images are still on the ProGet instance, but those shouldn't be the ones that end up getting used (I've been meaning to delete them). I've got a proxy set up that automatically redirects Docker traffic from `proget.makedeb.org/docker` and `proget.hunterwittenborn.com/docker` to `ghcr.io`.
> 
> Regardless of that, I've been wanting this to get done for a while. I have the proxy set up for people who need the old URLs, but GitHub Packages is where I'm wanting new users to start getting makedeb's images from.
> 
> Thanks for getting this done, but would you be available to discuss the issue with no being able to pull from the ProGet URLs? We could discuss it here or in makedeb's support rooms if you'd be open to that.

I can easily pull via `docker pull proget.hunterwittenborn.com/docker/makedeb/makedeb:ubuntu-jammy`. 

However when I try to build a dockerfile using `docker buildx build .` with a base image from `proget.*` then I get error message below.

```
ERROR: failed to solve: proget.hunterwittenborn.com/docker/makedeb/makedeb:ubuntu-jammy: failed to authorize: failed to fetch anonymous token: unexpected status: 403 Forbidden
```

Though, when I tried to build the same dockerfile in my personal computer, it didn't give any error. The reason was buildx was not installed, it was using the old `docker build`. When I intalled buildx, it gave the same error message.

In short 
✅ `docker pull proget.hunterwittenborn.com/docker/makedeb/makedeb:ubuntu-jammy`
❌ `docker buildx build .` which Dockerfile's base image is from proget.*
✅ `docker build .` which Dockerfile's base image is from proget.* and docker-buildx is not installed.

Probably there is a problem between `proget` and `docker buildx`.
